pkg/operator/controller/callbacks.go

```diff
@@ -355,3 +362,116 @@ func restoreOlderVersions(currentCrd, desiredCrd *extv1.CustomResourceDefinition
 	}
 	return desiredCrd
 }
+
+func reconcilePvcMutatingWebhook(args *callbacks.ReconcileCallbackArgs) error {
+	if args.State != callbacks.ReconcileStatePostRead {
+		return nil
+	}
+
+	deployment, ok := args.DesiredObject.(*appsv1.Deployment)
+	if !ok || deployment.Name != common.CDIApiServerResourceName {
+		return nil
+	}
+
+	enabled, err := featuregates.IsWebhookPvcRenderingEnabled(args.Client)
+	if err != nil {
+		return cc.IgnoreNotFound(err)
+	}
+
+	whc := &admissionregistrationv1.MutatingWebhookConfiguration{}
+	key := client.ObjectKey{Name: "cdi-api-pvc-mutate"}
+	err = args.Client.Get(context.TODO(), key, whc)
+	if err == nil {
+		if enabled {
+			return nil
```

Comment:
Can't exit here.   May have to update the webhook configuration when:

1) user changes any "critical" value
2) but perhaps most importantly, whenever the cacert gets rotated the cabundle will change

Reply:
fixed